src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.scss

```diff
@@ -0,0 +1,76 @@
+@import 'src/styles/common';
+
+.blastSubmissionResultsContainer {
+  max-width: 1860px;
+  padding: 0 $standard-gutter $global-padding-bottom 60px;
+}
+
+.sequenceBoxWrapper {
+  padding: 24px 26px; // TODO: the 26px padding looks like a constant asking to be extracted
```

Comment:
Why should `26px` be a constant/variable?

Reply:
I dont know, but this is something for latter to be done via a design review

Reply:
>> Why should 26px be a constant/variable?

> I dont know,

@imransl @jyothishnt 

See the same left padding in the header:

![image](https://user-images.githubusercontent.com/6834224/182551105-ce5146bb-b027-4000-bc91-68e70b5189b3.png)

as in the `.sequenceBoxWrapper`

![image](https://user-images.githubusercontent.com/6834224/182551153-848156c9-48ab-4199-9d0c-7a0056ed11ba.png)

And the same padding on the page that has a list of submissions:

![image](https://user-images.githubusercontent.com/6834224/182551728-749af2c2-e072-4d85-9d04-5b3beb43ad0c.png)